### ISSUE TITLE
Ability to see / reset to default (project) prefs

### ISSUE THREAD

**Issue by romw** [BOINC/boinc-ng#731]
**Reported by jbk on 28 Nov 38783582 07:33 UTC**
Some users report that they have played around with the prefs and subsequently forgotten what the default was. They request a way to reset prefs and project specific prefs or some way to see the default values for items in them.

Migrated-From: http://boinc.berkeley.edu/trac/ticket/754

**Comment by romw** [comment]
**Commented by jbk on 28 Mar 38783611 12:53 UTC**
This links somewhat to #607 (Unify prefs)

**Comment by Ageless93** [comment]
Why was this never picked up? We do have a "Default" button on the Event Log Options window, so why not on all the other windows (preferences, options) and web pages?

**Comment by CharlieFenton** [comment]
On the Manager's Computing Preferences dialog, the default is to use the web-based preferences instead. There is already a button for that.

**Comment by Ageless93** [comment]
I think what the users mean is a reset button to reset preferences to their initial state. Neither the local, nor the web preferences have this option.

**Comment by CharlieFenton** [comment]
I agree that the web preferences could use this button. But the initial / default state of the local preferences (on the Manager's Computing preferences dialog, before the user modifies it) is the web preferences. in other words, if there is no  global_prefs_override.xml file, then the Manager's dialog is populated with the values from global_prefs.xml, which has the values from the user's web preferences.

This is explained at the top of the Manager's Computing Preferences dialog which says either:
     Using web-based preferences from <project>
     Set values and click OK to use local preferences instead.
when there is no global_prefs_override.xml file, or:
     Using local preferences.
     Click "Use web prefs" to use web-based preferences from <project> 
when there is a global_prefs_override.xml file.

Given this behavior, what else would the "initial" or "default" settings of this dialog be?
